Wikibase/Sniffs/Commenting/ClassLevelDocumentationSniff.php

```diff
@@ -60,12 +62,31 @@ public function process( File $phpcsFile, $stackPtr ) {
 			return;
 		}
 
+		if ( $this->license ) {
+			$docClose = $phpcsFile->findPrevious( T_DOC_COMMENT_CLOSE_TAG, $stackPtr );
```

Comment:
At this point you can assume `$previous` is identical to `$docClose`. Essentially you can replace this line with `$docClose = $previous;`, saving an other (possibly expensive) find… call.

Reply:
It is in-memory array traversal - complexity O(n)

Reply:
Sure, and reusing a variable is essentially O(0).